packages/linux-driver-addons/dvb/crazycat/package.mk

```diff
@@ -7,8 +7,7 @@ PKG_SHA256="6b44a96d82c4a3e052864a995baceaede46b37c048c5718a6f62a009492d08ff"
 PKG_LICENSE="GPL"
 PKG_SITE="https://bitbucket.org/CrazyCat/media_build"
 PKG_URL="https://bitbucket.org/CrazyCat/media_build/get/$PKG_VERSION.tar.gz"
-PKG_DEPENDS_TARGET="media_tree_cc"
-PKG_NEED_UNPACK="$(get_pkg_directory media_tree_cc)"
+PKG_DEPENDS_UNPACK="media_tree_cc"
```

Comment:
I think want to keep `PKG_NEED_UNPACK`, the purpose of which is to automatically clean the `crazycat` package if the `media_tree_cc` package has changed.

With the changes above, we would unpack the new `media_tree_cc` package, but we wouldn't rebuild `crazycat` to use the new `media_tree_cc` changes.